internal/controller/config/config.go

```diff
@@ -69,8 +69,9 @@ func (*ConfigManager) GetControllerDefinitionsMgr() *NumaflowControllerDefinitio
 type GlobalConfig struct {
 	LogLevel          int    `json:"logLevel" mapstructure:"logLevel"`
 	IncludedResources string `json:"includedResources" mapstructure:"includedResources"`
-	// Feature flag - if enabled causes pipeline(s) to be paused when pipeline, numaflow controller, or ISB Service gets updated
-	DataLossPrevention bool `json:"dataLossPrevention" mapstructure:"dataLossPrevention"`
+	// If user's config doesn't exist or doesn't specify strategy, this is the default
+	// TODO: should we put this here or in usde config?
+	DefaultUpgradeStrategy USDEUserStrategy `json:"defaultUpgradeStrategy" mapstructure:"defaultUpgradeStrategy"`
```

Comment:
not sure if we want this to be in the USDE config instead

Reply:
sure. In the future, I think we should try to put most everything in our `numaplane-controller-config` - do you know why we have a separate file?

Reply:
this is not for the namespaced user upgrade strategy config but cluster wise for USDE config, right? then I'm not aware of reasons it has to be a separate file.

Reply:
right. In the future, we can try to just have the one file for stuff like this unless there's a good reason why it should be separate